src/styles/getTooltipStyles.js

```diff
@@ -93,6 +97,13 @@ export default function getTooltipStyles(
     const tooltipVerticalPadding = spacing.pv1;
     const tooltipFontSize = variables.fontSizeSmall;
 
+    // We get wrapper width based on tooltip's inner text width so the wrapper is just big enough to fit text and prevent white space
+    const wrapperWidth = tooltipTextWidth && tooltipTextWidth < maxWidth
+
```

Comment:
Can we get rid of this line? If the linter is mad and needs an empty line before a comment, maybe we can put the whole explanation of this calculation before line 101? I just don't really like the ternary 3 lines away from the original expression

Reply:
Done. I moved the comments before line 101. The ternary operation is close to expression.